prereise/gather/demanddata/bldg_electrification/zone_profile_generator.py

```diff
@@ -798,21 +873,80 @@ def main(zone_name, zone_name_shp, base_year, year, plot_boolean=False):
         "LADWP",
         "TID",
         "BANC",
-        "ISONE-Massachusetts",
+        "ISONE-Massachusetts",  # ISO-NE
         "ISONE-Maine",
         "ISONE-New Hampshire",
         "ISONE-Vermont",
         "ISONE-Connecticut",
         "ISONE-Rhode Island",
+        "PJM_AE",  # PJM
+        "PJM_AEP",
+        "PJM_AP",
+        "PJM_ATSI",
+        "PJM_BGE",
+        "PJM_ComEd",
+        "PJM_DAY",
+        "PJM_DEO&K",
+        "PJM_DLCO",
+        "PJM_DP&L",
+        "PJM_Dominion",
+        "PJM_EKPC",
+        "PJM_JCP&L",
+        "PJM_METED",
+        "PJM_PECO",
+        "PJM_PENELEC",
+        "PJM_PEPCO",
+        "PJM_PPL",
+        "PJM_PSEG",
+        "PJM_RECO",
+        "SPP-CSWS",  # SPP
+        "SPP-EDE",
+        "SPP-GRDA",
+        "SPP-KACY",
+        "SPP-KCPL",
+        "SPP-LES",
+        "SPP-MPS",
+        "SPP-NPPD",
+        "SPP-OKGE",
+        "SPP-OPPD",
+        "SPP-SECI",
+        "SPP-SPRM",
+        "SPP-SPS",
+        "SPP-WAUE",
+        "SPP-WFEC",
+        "SPP-WR",
+        "MISO-0001",  # MISO
+        "MISO-0004",
+        "MISO-0006",
+        "MISO-0027",
+        "MISO-0035",
+        "MISO-8910",
+        "PNM",  # Southwest
+        "EPE",
+        "TEPC",
+        "Arizona",
+        "WACM",
+        "PACE",
+        "PSCO",
+        "PSEI",  # Northwest
+        "DOPD",
+        "AVA",
+        "CHPD",
+        "GCPD",
+        "BPAT",
+        "PGE",
+        "PACW",
+        "IPCO",
+        "MT_west",
+        "NEVP",
+        "AECI",  # Southeast
+        "SOCO",
+        "AEC",
+        "TVA",
+        "Carolina",
+        "Florida",
```

Comment:
Should we put all constant lists used in main function into `const.py`, so that we can simply do `from const import zone_names, zone_name_shps` to increase the readability?

Reply:
Made the changes